src/version.h

```diff
@@ -31,13 +31,14 @@ will be invalid unless migration functions are written.
 #define REPOSITORY_FORMAT                                           5
 
 /***********************************************************************************************************************************
-Project version components. PROJECT_VERSION is automatically generated from the component parts.
+Project version components. PROJECT_VERSION and PROJECT_VERSION_NUM are automatically generated from the component parts.
 ***********************************************************************************************************************************/
 #define PROJECT_VERSION_MAJOR                                       2
 #define PROJECT_VERSION_MINOR                                       55
 #define PROJECT_VERSION_PATCH                                       0
 #define PROJECT_VERSION_SUFFIX                                      "dev"
 
 #define PROJECT_VERSION                                             "2.55.0dev"
+#define PROJECT_VERSION_NUM                                         2055000
```

Comment:
you're counting 1 digit for major, and 3 digits for minor and patch then?

Reply:
Yes -- for future proofing. We are already at 2.55.

Reply:
Also, there are many digits as we like to major version -- we just don't prepend 0s. You'll notice that I also changed version-num to a number in the JSON. This makes comparisons easier and is equivalent to what Postgres does.

Reply:
Cool. Thanks for the clarification :+1: